apps/back-office/src/app/(dashboard)/settings/team-members/components/AddTeamMember/LicenseInfoForm.tsx

```
@@ -0,0 +1,228 @@
+          )}
+        />
+
+        <form.Field
+          name="license.expirationDate"
+          validators={{
+            onBlur: ({ value }) => {
+              if (!value) return "Expiration date is required";
+              return undefined;
+            },
+          }}
+          children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)